test/FsAutoComplete.Tests.Lsp/CodeLensTests.fs

```diff
@@ -101,10 +101,23 @@ let tests state =
                   (args[1] :?> JObject).ToObject<Ionide.LanguageServerProtocol.Types.Position>(),
                   (args[2] :?> JArray) |> Seq.map (fun t -> (t:?>JObject).ToObject<Ionide.LanguageServerProtocol.Types.Location>()) |> Array.ofSeq
                 Expect.equal filePath doc.Uri "File path should be the doc we're checking"
-                Expect.equal triggerPos { Line = 1; Character = 8 } "Position should be 0:0"
+                Expect.equal triggerPos { Line = 1; Character = 6 } "Position should be 1:6"
```

Comment:
This changed because we stopped adding 2 to the column count as part of this fix. Why did we do that originally? Completely unknown to me reasons.